CI/physmon/phys_perf_mon.sh

```diff
@@ -27,7 +27,7 @@ function run() {
     histcmp \
         --label-reference=$refcommit \
         --label-monitored=$commit \
-        "$@"
+        "$@" || true
```

Comment:
Just out of curiosity, why is this needed if `set +e` is set?

Reply:
Because I used to collect the exit codes and combine them for the script exit code. I didn't want to completely remove that combination logic, and keep it for now, so I write 1 into `$ec` here. I could also do this differently, you're correct.